lib/cdo/scripts/script_info.rb

```diff
@@ -0,0 +1,54 @@
+# Info on existing tutorial scripts on Code Studio,
+# sometimes referenced in Pegasus.
+# Used for conditional behaviors.
+
+class ScriptInfo
```

Comment:
oh I think if you want to include if you'd have to make this a module not a class. A module probably makes more sense since you never instantiate it